truss/templates/trtllm-briton/src/engine.py

```diff
@@ -132,6 +135,29 @@ def load(self):
         briton_monitor_thread.start()
         self._loaded = True
 
+    def validate_input(self, prompt, model_input):
+        # Input length <= max_input_length.
+        if prompt:
+            input_length = len(prompt)
+            if input_length > self._max_input_len:
+                raise ValueError(
+                    f"Input length `{input_length}` is longer than allowed by max_input_length: {self._max_input_len}."
+                )
+
+        beam_width = model_input.get("beam_width", None)
+        if beam_width:
```

Comment:
better to do beam_width is not None

Reply:
nit the two conditions can be combined:
`if beam_width is not None and beam_width != 1`

Reply:
done and done.